src/libraries/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/ChaCha20Poly1305.NotImplemented.cs

```diff
@@ -0,0 +1,46 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
+
+namespace System.Security.Cryptography
+{
+    public partial class ChaCha20Poly1305
+    {
+        public static bool IsSupported => false;
+
+        private void ImportKey(ReadOnlySpan<byte> key)
```

Comment:
The `NotImplemented` partial should probably use the same name as the Browser one, after that gets renamed to NotSupported/Unsupported/etc.